lib/index.js

```diff
@@ -74,6 +75,7 @@ function innerRpc(request, callback) {
     port: self.port,
     rejectUnauthorized: self.rejectUnauthorized,
     agent: self.disableAgent ? false : undefined,
+    timeout: self.timeout,
```

Comment:
I would add it only in case it's defined. We don't know what the default value is.